CMakeLists.txt

```diff
@@ -78,16 +78,16 @@ set(OMR_GC         ON  CACHE INTERNAL "Enable the GC")
 set(OMR_FVTEST     OFF CACHE INTERNAL "Disable OMR's internal test suite, it's incompatible with b9")
 set(OMR_WARNINGS_AS_ERRORS OFF CACHE INTERNAL "OMR doesn't compile cleanly on my laptop :p")
 
-# Compile a b9-js *.src to C++
+# Compile a b9-js *.js (formerly, *.src) to C++
```

Comment:
I'm not sure documenting the old .src convention is really useful here. Might as well erase any mention and declutter the project.

Reply:
I don't know why I thought keeping that would be useful 😦